checklists/buddy.json

```diff
@@ -14,8 +14,8 @@
 		"dependsOn": ["dayZero"]
 	},
 	"findEmail": { 
-		"displayName": "Find email address", 
-		"description": "Find your new hire's email address.",
+		"displayName": “Get new hire’s email address", 
```

Comment:
Looks like there are a bunch of "smart" quotation marks in this file, which cause the JSON to be invalid. Please replace all the `“` and `”` with regular old `"`. That should fix the broken tests.